### DISCUSSION TITLE
Missing method getListColorChecker() in the CCheckerDetector Java class

### DISCUSSION THREAD

**Discussion by JeeDevUser** [bytedeco/javacpp#570] [General]
Hi,

Just tried a fresh snapshot, the method 
**getListColorChecker()**
https://docs.opencv.org/4.x/d9/d53/classcv_1_1mcc_1_1CCheckerDetector.html#a035f9ea8c21a9e7d160b622ca94c71f2

seems to be missing from the CCheckerDetector Java class:

http://bytedeco.org/javacpp-presets/opencv/apidocs/org/bytedeco/opencv/opencv_mcc/CCheckerDetector.html

Can anyone check that?

**Comment by saudet** [comment]
Duplicate of #571

**Reply by JeeDevUser** [reply]
Yeah, true, I don't see how to delete this message, however, so that 
https://github.com/bytedeco/javacpp/issues/571
 remains alone...
